spanner/CHANGES.md

```diff
@@ -1,5 +1,39 @@
 # Changes
 
+## v1.11.0
+
+* Features:
+  - feat(spanner): add KeySetFromKeys function (#2837)
+* Misc:
+  - test(spanner): check for Aborted error (#3039)
+  - test(spanner): fix potential race condition in TestRsdBlockingStates (#3017)
+  - test(spanner): compare data instead of struct (#3013)
+  - test(spanner): fix flaky oc_test.go (#2838)
+  - docs(spanner): document NULL value (#2885)
+* spansql/spannertest:
+  - feat(spanner/spansql): parse join hints (#2936)
+  - feat(spanner/spansql): parse path expressions (#2924)
+  - feat(spanner/spansql): parse joins (#2896)
+  - feat(spanner/spannertest): implement RIGHT JOIN (#3042)
+  - feat(spanner/spannertest): support INNER and CROSS JOIN (#3037)
+  - feat(spanner/spannertest): support USING clause in JOIN (#2995)
+  - feat(spanner/spannertest): support LEFT OUTER JOIN with ON clause (#2945)
+  - feat(spanner/spannertest): evaluate path expressions (#2931)
```

Comment:
I think this can be collapsed into a single note, of supporting JOINs (all but FULL JOIN).